### ISSUE TITLE
`x.modules.import.imPar` vs `imPar`

### ISSUE THREAD

**Issue by HenkMutsaerts** [ExploreASL/ExploreASL#931]
### Description

We use `x.modules.import.imPar` and `imPar`, which is confusing, can we choose one of these?
Probably best to only use `x.modules.import.imPar`.

### Requirements

See `xASL_module_Import` line 146, and throughout other functions

### How to test

Optional: insert description about how to test the code changes here

### Release notes

Required: summarize the changes for the release notes here

**Comment by MichaelStritt** [comment]
Yeah, we already started moving everything to the x structure. It's just an incremental process where we're trying to keep everything stable in the meantime.
